src/_nebari/stages/kubernetes_services/__init__.py

```diff
@@ -518,6 +519,7 @@ def input_vars(self, stage_outputs: Dict[str, Dict[str, Any]]):
                 self.config.default_images.dask_worker
             ),
             dask_gateway_profiles=self.config.profiles.dict()["dask_worker"],
+            cloud_provider=cloud_provider,
```

Comment:
@marcelovilla is this needed?

Reply:
I'm assuming it's needed as we want to pass it to the dask_gateway module, right? Just like we're passing the cloud provider to the jupyterhub module in https://github.com/nebari-dev/nebari/blob/195738ef77f4fae7bc34720d4e1e613bf273b55b/src/_nebari/stages/kubernetes_services/__init__.py#L370

Let me know if this makes sense

Reply:
you are correct, but on that line we are passing that same variable to `MonitoringInput` which I think goes to grafana

Reply:
I see what is going on. The diff view only shows some portions of the code and in this case if you don't expand the code, it looks like it's under the MonitoringInputVars but it's really not.